ui/src/test/NewInventory/__snapshots__/NewInventory.test.tsx.snap

```diff
@@ -0,0 +1,280 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Does this snapshot have value?  I would think that this would fail frequently simple because we are still developing quickly.  I worry that failed snapshots would be ignored because we expect them to fail.  In Controller we abandoned using snapshot tests for this reason.

Reply:
@AlexSCorey - I have only added the snapshots testing as the first round of tests - was planning to replace them with more targetd tests later. Will not change the tests now, as the UI will move to monorepo.